README.md

```diff
@@ -11,3 +11,9 @@ which will get Upstart out of the confused state.
 When “`status <jobname>`” says “`<jobname> stop/killed, process 12345`” and
 there’s no such process, run “`workaround-upstart-snafu 12345`” and wait until
 it exits.
+
+To speed things up (i.e. to limit the PID space this script covers), issue the following command:
+
+    echo $N | sudo tee /proc/sys/kernel/pid_max
```

Comment:
I'm not sure if/how I can amend your branch in order to fix this PR, but you should get the credit for contributing, so I'll let you make the changes, (unless you want me to create my own PR) :)